cpu/sam0_common/include/timer_config.h

```diff
@@ -0,0 +1,221 @@
+#endif
+#ifdef MCLK_APBBMASK_TC5
+#define MCLK_TC5      (&MCLK->APBBMASK.reg)
+#define MCLK_TC5_MASK (MCLK_APBBMASK_TC5)
+#endif
+#ifdef MCLK_APBCMASK_TC5
+#define MCLK_TC5      (&MCLK->APBCMASK.reg)
+#define MCLK_TC5_MASK (MCLK_APBCMASK_TC5)
+#endif
+#ifdef MCLK_APBDMASK_TC5
+#define MCLK_TC5      (&MCLK->APBDMASK.reg)
+#define MCLK_TC5_MASK (MCLK_APBDMASK_TC5)
+#endif
+
+#ifdef MCLK_APBAMASK_TC6
+#define MCLK_TC6      (&MCLK->APBAMASK.reg)
+#define MCLK_TC6_MASK (MCLK_APBAMASK_TC6)
+#endif
+#ifdef MCLK_APBBMASK_TC6
+#define MCLK_TC6      (&MCLK->APBBMASK.reg)
+#define MCLK_TC6_MASK (MCLK_APBBMASK_TC6)
+#endif
+#ifdef MCLK_APBCMASK_TC6
+#define MCLK_TC6      (&MCLK->APBCMASK.reg)
+#define MCLK_TC6_MASK (MCLK_APBCMASK_TC6)
+#endif
+#ifdef MCLK_APBDMASK_TC6
+#define MCLK_TC6      (&MCLK->APBDMASK.reg)
+#define MCLK_TC6_MASK (MCLK_APBDMASK_TC6)
+#endif
+
+#ifdef MCLK_APBAMASK_TCC0
+#define MCLK_TCC0      (&MCLK->APBAMASK.reg)
+#define MCLK_TCC0_MASK (MCLK_APBAMASK_TCC0)
+#endif
+#ifdef MCLK_APBBMASK_TCC0
+#define MCLK_TCC0      (&MCLK->APBBMASK.reg)
+#define MCLK_TCC0_MASK (MCLK_APBBMASK_TCC0)
+#endif
+#ifdef MCLK_APBCMASK_TCC0
+#define MCLK_TCC0      (&MCLK->APBCMASK.reg)
+#define MCLK_TCC0_MASK (MCLK_APBCMASK_TCC0)
+#endif
+#ifdef MCLK_APBDMASK_TCC0
+#define MCLK_TCC0      (&MCLK->APBDMASK.reg)
+#define MCLK_TCC0_MASK (MCLK_APBDMASK_TCC0)
+#endif
+
+#ifdef MCLK_APBAMASK_TCC1
+#define MCLK_TCC1      (&MCLK->APBAMASK.reg)
+#define MCLK_TCC1_MASK (MCLK_APBAMASK_TCC1)
+#endif
+#ifdef MCLK_APBBMASK_TCC1
+#define MCLK_TCC1      (&MCLK->APBBMASK.reg)
+#define MCLK_TCC1_MASK (MCLK_APBBMASK_TCC1)
+#endif
+#ifdef MCLK_APBCMASK_TCC1
+#define MCLK_TCC1      (&MCLK->APBCMASK.reg)
+#define MCLK_TCC1_MASK (MCLK_APBCMASK_TCC1)
+#endif
+#ifdef MCLK_APBDMASK_TCC1
+#define MCLK_TCC1      (&MCLK->APBDMASK.reg)
+#define MCLK_TCC1_MASK (MCLK_APBDMASK_TCC1)
+#endif
+
+#ifdef MCLK_APBAMASK_TCC2
+#define MCLK_TCC2      (&MCLK->APBAMASK.reg)
+#define MCLK_TCC2_MASK (MCLK_APBAMASK_TCC2)
+#endif
+#ifdef MCLK_APBBMASK_TCC2
+#define MCLK_TCC2      (&MCLK->APBBMASK.reg)
+#define MCLK_TCC2_MASK (MCLK_APBBMASK_TCC2)
+#endif
+#ifdef MCLK_APBCMASK_TCC2
+#define MCLK_TCC2      (&MCLK->APBCMASK.reg)
+#define MCLK_TCC2_MASK (MCLK_APBCMASK_TCC2)
+#endif
+#ifdef MCLK_APBDMASK_TCC2
+#define MCLK_TCC2      (&MCLK->APBDMASK.reg)
+#define MCLK_TCC2_MASK (MCLK_APBDMASK_TCC2)
+#endif
+
+#ifdef MCLK_APBAMASK_TCC3
+#define MCLK_TCC3      (&MCLK->APBAMASK.reg)
+#define MCLK_TCC3_MASK (MCLK_APBAMASK_TCC3)
+#endif
+#ifdef MCLK_APBBMASK_TCC3
+#define MCLK_TCC3      (&MCLK->APBBMASK.reg)
+#define MCLK_TCC3_MASK (MCLK_APBBMASK_TCC3)
+#endif
+#ifdef MCLK_APBCMASK_TCC3
+#define MCLK_TCC3      (&MCLK->APBCMASK.reg)
+#define MCLK_TCC3_MASK (MCLK_APBCMASK_TCC3)
+#endif
+#ifdef MCLK_APBDMASK_TCC3
+#define MCLK_TCC3      (&MCLK->APBDMASK.reg)
+#define MCLK_TCC3_MASK (MCLK_APBDMASK_TCC3)
+#endif
+/** @} */
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif /* TIMER_CONFIG_H */
+/** @} */
```

Comment:
What's the point of this file ?

Reply:
To allow for a generic `TCC_CONFIG` (and later `TC_CONFIG`) macro.